### DISCUSSION TITLE
MSFS 2024 Controls

### DISCUSSION THREAD

**Discussion by ricardo404** [redphx/better-xcloud#565] [General]
Has anyone been able to get the keyboard to work in 2024, as it did in 2020? At least for me here at Xcloud the controls are not working properly, as if there is some conflict. Anyone else with that?

**Comment by Sus002** [comment]
same for me, it works and its recognized by the simulator, the keyboard works normally but the mouse does not, im hoping red can somehow add native mkb support like in 2020

**Reply by ricardo404** [reply]
> mesmo para mim, funciona e é reconhecido pelo simulador, o teclado funciona normalmente, mas o mouse não, espero que o vermelho possa de alguma forma adicionar suporte nativo a mkb como em 2020

I noticed that keyboard detection was removed in msfs2024 on Xcloud (I believe due to the bug I was having between the keyboard and the controller). Can you still use the keyboard using better xcloud?

**Comment by pdario** [comment]
I'm trying this (https://github.com/redphx/better-xcloud/discussions/574) but both M&K are dead for me (on Edge)
